src/components/NavBar.tsx

```diff
@@ -28,8 +32,12 @@ const NavBar: React.FC = () => {
   return(
     
     <ul className={classes.navbar}>
-      <li className={clsx(classes.navbarElement, classes.active)}><a href=''>Home</a></li>
-      <li className={clsx(classes.navbarElement)}><a href=''>Studio</a></li>
+      <li className={clsx(classes.navbarElement, classes.active)}>
+        <Link to="/" className={classes.tab}>
+          Home 
+          <img style={{width: 20, height: 20, paddingLeft: 5}} src='images/HomeImage.png' alt=''></img>
```

Comment:
@see previous comment about units

Reply:
Also better to store image locations are constants at top of the file. It makes it easier to change/update in the future without searching through the entire file